tests/MongoDB.Driver.Core.Tests/Core/Operations/OperationTestBase.cs

```diff
@@ -247,9 +248,10 @@ protected IReadBinding CreateReadBinding(ReadPreference readPreference)
             return new ReadPreferenceBinding(_cluster, readPreference, _session.Fork());
         }
 
-        protected IReadWriteBinding CreateReadWriteBinding()
+        protected IReadWriteBinding CreateReadWriteBinding(bool useExplicitSessionIfSupported = true)
```

Comment:
Let's be consistent and use "implicit" everywhere:

    protected IReadWriteBinding CreateReadWriteBinding(bool useImplicitSession = false)
    {
        var options = new CoreSessionOptions(isImplicit: useImplicitSession);
        var session = CoreTestConfiguration.StartSession(_cluster, options);
        return new WritableServerBinding(_cluster, session);
    }

Reply:
Suggestion above requires slight change to `CoreTestConfiguration.StartSession` to add `options` parameter:

    public static ICoreSessionHandle StartSession(ICluster cluster, CoreSessionOptions options = null)
    {
        if (AreSessionsSupported(cluster))
        {
            return cluster.StartSession(options);
        }
        else
        {
            return NoCoreSession.NewHandle();
        }
    }